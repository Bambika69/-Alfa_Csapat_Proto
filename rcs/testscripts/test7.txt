createfield -t laboratory
createfield L1 -t safehouse
createvirologist SA1
creategeneticcode bear -l L1
createequipment cloak -v V1
setrandom False random
startturn V1
toggle E1
move L1