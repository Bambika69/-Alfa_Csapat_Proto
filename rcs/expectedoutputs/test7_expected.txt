Field created:
ID: L1
Type: Laboratory
Neighbours: null

Field created:
ID: F1
Type: Field
Neighbours: L1

Virologist created:
ID: V1
Field: F1
Backpack: B1

GeneticCode created:
ID: BC1
Type: Bear
Destination: L1

Equipment created:
ID: C1 
Type: Cloak
Destination: V1
Result: successful

Random behavior changed:
yesOrNoType: False
choiceType: random

V1â€™s turn started.

Equipment toggled on Virologist:
Equipment: C1
Virologist: V1
Inf: wear

Virologist moved: 
Virologist: V1
Destination: L1

Agent created:
ID: BVI1
Type: virus
GeneticCode: Bear
Virologist: V1
Result: successful

Agent used on Virologist:
Agent: BVI1
Virologist: V1
Target: V1
Result: successful
